boefjes/boefjes/plugins/kat_adr_finding_types/boefje.json

```diff
@@ -1,7 +1,7 @@
 {
   "id": "adr-finding-types",
   "name": "ADR Finding Types",
-  "description": "Hydrate information of ADR finding types",
+  "description": "Checks identified API Design Rules (ADR) for common design mistakes.",
```

Comment:
This is not really what the boefje does.

Reply:
This was an old version before your explanation, the new version is now submitted.